data/json/recipes/recipe_electronics.json

```diff
@@ -158,7 +158,8 @@
     [
       [ "steel_chunk", 1 ],
       [ "scrap", 3 ],
-      [ "knife_butter", 2 ]
+      [ "knife_butter", 2 ],
+      [ "material_aluminium_ingot", 1 ]
```

Comment:
90% of mass is lost. Also nobody will use this recipe given how available the other items are